src/AzureRMAssets.ts

```diff
@@ -15,6 +15,7 @@ export class AzureRMAssets {
 
     // For test dependency injection only
     public static setFunctionsMetadata(metadataString: string): void {
+        // tslint:disable-next-line:typedef
```

Comment:
Why disable this rule rather than type defining the property??

Reply:
Because I hate providing obvious stuff that the compiler is already smart enough to infer anyway.